charts/console/templates/platform-cortex/deployment.yaml

```diff
@@ -0,0 +1,190 @@
+{{- if .Values.platformCortex.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ include "conduktor.platformCortex.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: conduktor-platform-cortex
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: {{ .Values.platformCortex.replicaCount }}
+  {{- if .Values.platformCortex.updateStrategy }}
+  strategy: {{- toYaml .Values.platformCortex.updateStrategy | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: conduktor-platform-cortex
+  template:
+    metadata:
+      annotations:
```

Comment:
This annotations section is missing `commonAnnotations`

Reply:
Yes and platform deployment too it seems, will fix that :+1: